plugins/SLURM-Integration/ufm-epilog.sh

```diff
@@ -12,4 +12,4 @@
 #
 SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
 cd $SCRIPTPATH
-python3 ufm_slurm_epilog.py -i $SLURM_JOBID
+sudo python3_path ufm_slurm_epilog.py -i $SLURM_JOBID
```

Comment:
why need sudo? AFAIR prolog/epilog is executed as root

Reply:
fixed